src/index.ts

```diff
@@ -1,17 +1,23 @@
 import { Client } from 'rpc-websockets'
 
 const CONDUCTOR_CONFIG = '/_dna_connections.json'
+const DEFAULT_TIMEOUT = 5000
```

Comment:
It'd be really helpful to the Apollo LInk adapter I'm building if we also had a `DEFAULT_RETRY_ATTEMPTS` (or otherwise just make this an option object to pass through to rpc-websockets client config. In the Apollo Link setup I need retry attempts to be 0 so that Apollo-Link Retry can do it's magic. Hope that makes sense, this is what this issue is about: https://github.com/holochain/hc-web-client/issues/24

Reply:
@lorenjohnson yeah makes sense -- what if we just add an additional param to `connect` which would be the config object used to construct the rpc-websockets `Client`? That includes max_reconnects and reconnect_interval. https://github.com/elpheria/rpc-websockets/blob/master/src/lib/client.js#L24

So you could be like:

```js
const foo = await connect({
    timeout: 5000,
    wsOpts: {
        reconnect: false,
        max_reconnects: 0,
    }
})
```

Reply:
Yeah, I think that'd be perfect. 

It'd be good to be mindful that the remaining option defaults for rpc-websockets not described by options sent in `wsOpts` be retained. Probably the default behavior if the opts are mostly passed through, but felt worth mentioning.